Description: Set default paths for operation of program operation.
 Change default log path from /var/spool/log to /var/log.
 Change default path for .chs files from /home/fido/jamnntpd/xlat to their
 installation directory /usr/share/jamnntpd/xlat.
Author: Robert James Clay <jame@rocasa.us>
Origin: vendor
Last-Update: 2014-05-20

--- a/src/os_linux.h
+++ b/src/os_linux.h
@@ -14,7 +14,7 @@
 #define PLATFORM_NAME "Linux"
 
 #define CFG_BASEPATH "/etc/"
-#define LOG_BASEPATH "/var/spool/log/"
+#define LOG_BASEPATH "/var/log/"
 
 #define TRUE 1
 #define FALSE 0
--- a/examples/jamnntpd.xlat
+++ b/examples/jamnntpd.xlat
@@ -8,7 +8,7 @@
 
 # This is the directory where the *.chs files will be found
 
-chsdir /home/fido/jamnntpd/xlat
+chsdir /usr/share/jamnntpd/xlat
 
 # Charset translations when reading messages
 
--- a/unicode/jamnntpd.xlat
+++ b/unicode/jamnntpd.xlat
@@ -4,7 +4,7 @@
 # will be posting in or JamNNTPd will not be able to translate the receiver 
 # name properly. Unfortunately posting in utf-8 is not possible at this time.
 
-chsdir /home/fido/jamnntpd/xlat
+chsdir /home/share/jamnntpd/xlat
 
 defaultread IBMPC
 
